openapi.yaml

```diff
@@ -22947,6 +23326,10 @@ components:
               type: boolean
               description: If true, this User may add VPCs.
               example: true
+            child_account_access:
+              type: boolean
+              description: In a [parent and child account](/docs/guides/parent-child-accounts/) environment, this gives a parent account access to endpoints that can be used to manage child accounts. Unrestricted parent account users have access to this grant, while restricted parent users don't. (An unrestricted parent user can set this to `true` to add this grant to a restricted parent user.)
+              example: false
```

Comment:
@afolksetapart ... This wasn't included in the API spec. So, I'm just guessing that the `child_account_access` grant has to have been added to endpoints that show/use user grants, correct?

Reply:
Sorry about that Chris - yes that's correct. It will be `null` for everyone except parent account users, and only parent account users will be able to set it. If an account goes from being a parent account, to not being a parent account, to being a parent account again, this grant will be "reinstated" for whoever had it before the transition.